Design Twitter.java

```diff
@@ -0,0 +1,113 @@
+// Time Complexity :
+// Space Complexity :
+// Did this code successfully run on Leetcode :No
+// Any problem you faced while coding this :
+
+
+// Your code here along with comments explaining your approach
+
+class Twitter {
+  int timestamp;
+  int feedsize;
+ HashMap< Integer,List<Tweet>> tweetsmap;
+  HashMap<Integer,Set<Integer>> followersmap;
+  List<Tweet> getfeed;
+  /** Initialize your data structure here. */
+  public Twitter() {
+      timestamp=0;
+      feedsize =10;
+      tweetsmap=new HashMap<>();
+      followersmap=new HashMap<>();
+
+  }
+  
+  /** Compose a new tweet. */
+  public void postTweet(int userId, int tweetId) {
+      isFirsttime(userId);
+      List<Tweet> t=tweetsmap.get(userId);
+      t.add(new Tweet(tweetId,timestamp++));
+  }
+  
+  /** Retrieve the 10 most recent tweet ids in the user's news feed. Each item in the news feed must be posted by users who the user followed or by the user herself. Tweets must be ordered from most recent to least recent. */
+  public List<Integer> getNewsFeed(int userId) {
+       isFirsttime(userId);
+      getfeed= new ArrayList<>();
+      Set<Integer> users=followersmap.get(userId);
+      for(Integer i:users){
+          getfeed.addAll(tweetsmap.get(i));
+      }
+      Collections.sort(getfeed,(a,b) ->(b.CreatedAt -a.CreatedAt));
+      
+      List<Integer> recentfeed =new ArrayList<>();
+      Iterator<Tweet> itr=getfeed.iterator();
+      int count=feedsize;
+      while(count>0 && itr.hasNext()){
+          recentfeed.add(itr.next().TweetId);
+          count--;
+      }
+      
+      
+      return recentfeed;
+  }
+  
+  /** Follower follows a followee. If the operation is invalid, it should be a no-op. */
+  public void follow(int followerId, int followeeId) {
+     isFirsttime(followerId);
+       isFirsttime(followeeId);
+      Set<Integer> f=followersmap.get(followerId);
+      f.add(followeeId);
+  }
+  
+  /** Follower unfollows a followee. If the operation is invalid, it should be a no-op. */
+  public void unfollow(int followerId, int followeeId) {
+        isFirsttime(followerId);
+       isFirsttime(followeeId);
+      if(followerId != followeeId){
+           followersmap.remove(followerId);
```

Comment:
There is a bug here. `followersmap.remove(followerId);`
replace this line of code with 
` Set<Integer> followees = followersmap.get(followerId);
           followees.remove(followeeId);`

Basically you were not getting set of followees. You were remove follower itself from the followersmap.